addon/components/frost-link.js

```diff
@@ -0,0 +1,106 @@
+import Ember from 'ember'
+import layout from '../templates/components/frost-link'
+
+function addPriorityClass (priority, classes) {
+  switch (priority) {
+    case 'primary':
+      classes.push('primary')
+      break
+    case 'secondary':
+      classes.push('secondary')
+      break
+    case 'tertiary':
+      classes.push('tertiary')
+      break
+    default:
+      // no class to add for invalid priority
+      break
+  }
+}
+
+function addSizeClass (size, classes) {
+  switch (size) {
+    case 'small':
+      classes.push('small')
+      break
+    case 'medium':
+      classes.push('medium')
+      break
+    case 'large':
+      classes.push('large')
+      break
+    default:
+      // no class to add for invalid size
+      break
+  }
+}
+
+export default Ember.LinkComponent.extend({
+  classNames: ['frost-link'],
+
+  classNameBindings: [
+    'disabled',
+    'extraClasses',
+    'isPressed'
```

Comment:
This is the :active CSS selector - we don't need to detect mouse up/down ourselves